### PR TITLE
Translation maintaining balance into es

### PR DESCRIPTION
**Pull request by RestrepoTorres**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Comment by bw4cnc** [comment]
I don't understand GitHub. I don't know what to do. This like alot of of
work to understand.   Not sure what I'm suppose to do. Should I stop using
GitHub?

Blake Williamson

**Comment by bw4cnc** [comment]
No I haven't because don't understand.

Blake Williamson

File: Gemfile [hunk 1]
```diff
@@ -5,4 +5,4 @@ gem "github-pages", group: :jekyll_plugins
 group :test do
   gem "html-proofer", "~> 3.19.4"
   gem "rake"
-end
+end
```
Comment (underwoo16) [inline Gemfile:8]: Please remove change to Gemfile

**Review by underwoo16 — CHANGES_REQUESTED** [review]
Hello, thanks for the contribution!

The translated article looks great, I am requesting that you remove changes to any other files

**Pull request closed without merging**